docs/instructions.md

```diff
@@ -62,16 +79,94 @@ env = android_env.load(
 )
 ```
 
-## Example scripts
+## Example RL agent scripts
+
+The `examples` directory contains a few simple example agent setups, such as:
+
+*   [`run_random_agent.py`](https://github.com/deepmind/android_env/blob/main/examples/run_random_agent.py):
+    Runs a simple loop performing randomly selected actions in the environment.
+*   [`run_acme_agent.py`](https://github.com/deepmind/android_env/blob/main/examples/run_acme_agent.py):
+    Runs a training loop with an [Acme](https://deepmind.com/research/publications/Acme)
+    DQN agent, implemented in the popular DeepMind RL framework. This will
+    require to install the [`acme`](https://github.com/deepmind/acme)
+    dependency.
+*   [`run_human_agent.py`](https://github.com/deepmind/android_env/blob/main/examples/run_human_agent.py):
+    Creates a [`pygame`](https://www.pygame.org) instance that lets a human user
+    interact with the environment and observe environment mechanics, such as
+    rewards or task extras. You will need to install the [PyGame] dependency.
+
+For instance, here is how you can run `run_random_agent.py` in a folder where
+you have your APK file, such as
+the [Apple Flinger](https://github.com/deepmind/android_env/blob/main/docs/example_tasks.md#apple-flinger)
+game.
+(You can find the TAR file with the APK and `.textproto` files
+[here](https://storage.googleapis.com/android_env-tasks/apple_flinger.tar.gz)
```

Comment:
Can you remove the specific links to the tar file? If in the future we put these in other locations, it'd be easier to change them in a single place. Maybe you can say something like "The downloaded task contains the `APK` and `.textproto` definitions that you can use for this example"?

Reply:
Done! Thank you @kenjitoyama